feat(onboarding): joining the company (#21)
zatsit-oss/zats-blog-content: blog/general/2024-07-16-onboarding/index.md

---
slug: onboarding

> title: Mon onboarding

Reviewer: ```suggestion
title: Mon onboarding chez zatsit
```

Author: Done

authors: [jvasseur]
tags: [general]
---

Mon arrivée dans une nouvelle aventure !

<!-- truncate -->

![Gourde  et sweat](gourde_sweat_banniere.webp)

> Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien et Océane. Un bureau m'attend avec quelques goodies estampillés au couleur de l'entreprise, gourde, sweat, sticker et un petit mot d'accueil qui me font très plaisir. 

Reviewer: ```suggestion
Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de mission RH de choc). Un bureau m'attend avec quelques goodies estampillés au couleur de l'entreprise : gourde, sweat, sticker et un petit mot d'accueil qui me font très plaisir. 
```

Author: Done

> On commence par gérer tout ce que tout le monde « adore »… la partie administrative et tous les divers documents à fournir et quelques validations d’identité habituelles. On active mes accès à [Ekip](https://www.ekip.app/) ainsi je découvre les tickets restaurants écoresponsables. Puis avec Océane on passe en revue les accès aux outils qui permettent le suivi pour mes futures missions.

Reviewer: ```suggestion
La matinée commence par la partie administrative ( ce que tout le monde « adore », n'est ce pas ? )
- les divers documents à fournir et quelques validations d’identité habituelles. 
- On active mes accès à [Ekip](https://www.ekip.app/). Je découvre ainsi les tickets restaurants **éco-responsables**. 
- Puis avec Océane, on passe en revue les accès aux outils qui permettent le suivi pour mes futures missions. D'ailleurs, **zatsit** a mis en place une plateforme dédiée pour l'onboarding des consultants, permettant mettre en avant les quelques « formations  » à dérouler pour être opérationnel.
```

Author: Done

> On reparle avec tous le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir [Zatsit](https://zatsit.fr/) sur les aspects sociétal et environnemental ce qui m’amène à vous parler d’un choix sur le matériel qu’a fait la société...  
> 

Reviewer: ```suggestion
On reparle avec tous le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir **[zatsit](https://zatsit.fr/)** sur les aspects sociétal et environnemental ; Ce qui m’amène à vous parler du choix sur le matériel qu’a fait la société.

```

Author: Done

> ## Maintenant le plus cool : mon poste de travail...

Reviewer: ```suggestion
## Mon meilleur moment de la journée : la découverte de mon outil de travail
```

Author: Done

> Peu de temps avant mon arrivée, j’ai reçu un message qui me demandait de faire le choix de mon futur poste de travail, ainsi j’ai découvert le concept de [Framework](https://frame.work/fr/fr) ce qui permet à Zatsit pour être cohérent dans sa politique environnementale et avoir un impact le plus restreint possible sur la planête. J’ai donc choisi les différents éléments de ma machine 13 pouces avec coque en aluminium : processeur, ram, disque, extensions… Et pour le fun en DIY (Do It Yourself).

Reviewer: ```suggestion
Peu de temps avant mon arrivée, j’ai reçu un message qui me demandait de choisir la configuration de mon futur poste de travail [Framework](https://frame.work/fr/fr). 
J'ai ainsi découvert leur concept : proposer du matériel réparable et évolutif. Ce choix permet à **zatsit** de rester cohérent dans sa politique environnementale et avoir un impact le plus restreint possible sur la planête. 

J’ai donc choisi les différents éléments de ma machine : un 13 pouces avec coque en aluminium : processeur, ram, disque et extensions… Et pour le fun, le montage à faire par mes soins pour prendre en main l'assemblage du hardware.
```

Author: Done

![Assemblage framework](framework_assembling.webp)

> Donc tout était là, un carton contenant les diverses parties de mon ordinateur, quand on l’habitude de monter des configurations et comme tout est suffisamment indiqué sur la carte mère, on installe le disque SSD, la mémoire on referme avec le couvercle qui est le clavier, on serre quelque vis avec l’outil qui est fournit, quelques minutes suffises. Le petit truc sympa, le contour de l’écran est amovible et il existe plusieurs couleurs, moi j’ai choisi orange.

Reviewer: ```suggestion
Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement en en quelques minutes: on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil fournit, et *zatsit* ! 

Le petit plus sympa : le contour de l’écran est amovible et il existe plusieurs couleurs. Moi, j’ai choisi le orange.
```

Author: Done

![Extensions](extensions.webp)

> Pour pouvoir alimenter la machine et la connecter à divers équipements extérieurs, il y a 4 emplacements qui peuvent recevoir des modules d’extension en tous genres : USB type A ou C, HDMI, Ethernet et autres que l'on peut interchangé à chaud. Et la aussi on peut y ajouter un peu de couleur.

Reviewer: ```suggestion
Pour pouvoir alimenter la machine et la connecter à des périphériques, il y a 4 emplacements qui peuvent recevoir des modules d’extension en tous genres : USB type A ou C, HDMI, Ethernet et autres que l'on peut donc interchanger, et à chaud. 

Et pour certaines extensions, on peut également y ajouter un peu de couleur.
```

Author: Done

![Setup](setup.webp)

> Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB que j’avais préparé en avance en suivant les indications fournies sur le site du fabriquant. Et tout était fonctionnel au premier démarrage ! J’ai commencé à travailler sur ce poste après avoir configuré un environnent de développement. Je suis enthousiaste à l’idée d'utiliser ma machine lors de mes missions.

Reviewer: indications fournies sur le site du fabriquant -> mettre le lien vers la doc

```suggestion
Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB que j’avais préparé en avance en suivant les [indications fournies sur le site du fabriquant](lien ici). 

Tout était fonctionnel au premier démarrage ! 

J’ai commencé à travailler sur ce poste après avoir configuré mon environnent de développement. Je suis déjà enthousiaste à l’idée d'utiliser ma machine au quotidien chez nos clients.
```

Author: Done

> En parcourant le site de Framework [Framework](https://frame.work/fr/fr) on y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclé. Qu’il est aussi possible de commander tous les composants séparément ce qui permet de réparer ou de faire évoluer sa configuration facilement. Idée que je trouve intéressante quand on à déjà ouvert un ordinateur portable et de constater que l’on ne pourra pas faire grand chose pour se dépanner.

Reviewer: ```suggestion
Parcourir le site de [Framework](https://frame.work/fr/fr) est intéressant : 
- On y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclé. 
- Qu’il est aussi possible de commander tous les composants séparément, ce qui permet de réparer ou de faire évoluer sa configuration facilement au fil des années (Idée que je trouve intéressante, quand on a déjà ouvert un ordinateur portable et constaté que l’on ne pourra pas faire grand chose pour le dépanner)
- Chaque composant proposé sur la Markerplace possède sa fiche de réparabilité (dans l'idée) + mettre photo ?
```

Author: Done

> Il reste juste à voir dans le temps comment tout cela va évoluer et peut-être partager mon expérience dans quelques temps sur ce genre de machine. 

Reviewer: pas sur que cette ligne est sa place en conclusion d'article :) 

Pourquoi ne pas ajouter un chapitre sur tes semaines en interne avec le déroulé des classroom sur la tech chez nous ?

Par contre je pense qu'il faudrait conclure sur `la suite chez zatsit` non ? comment tu pressent les prochains mois, les sujets internes et externe sur lesquels tu va bosser, la qualité des missions qu'on t'as proposé, etc

Author: Done

---
slug: onboarding
title: Mon onboarding chez zatsit
authors: [jvasseur]
tags: [general]
---

Mon arrivée dans une nouvelle aventure !

<!-- truncate -->

![Gourde  et sweat](gourde_sweat_banniere.webp)

> Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de mission RH de choc). Un bureau m'attend avec quelques goodies estampillés au couleur de l'entreprise : gourde, sweat, sticker et un petit mot d'accueil qui me font très plaisir. 

Reviewer: ```suggestion
Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de mission RH de choc). Un bureau m'attend avec quelques goodies estampillés aux couleurs de l'entreprise : gourde, sweat, sticker et un petit mot d'accueil qui me font très plaisir. 
```

> la matinée commence par la partie administrative (ce que tout le monde « adore », n'est ce pas)

Reviewer: ```suggestion
La matinée commence par la partie administrative (ce que tout le monde « adore », n'est ce pas ?)
```

- les divers documents à fournir et quelques validations d’identité habituelles.
- On active mes accès à [Ekip](https://www.ekip.app/). Je découvre ainsi les tickets restaurants **écoresponsables**.
- Puis avec Océane on passe en revue les accès aux outils qui permettent le suivi pour mes futures missions. D'ailleurs, **zatsit** a mis en place une plateforme dédiée pour l'onboarding des consultants, permettant de mettre en avant les quelques « formations » a dérouler pour être opérationnel. 

> On reparle avec tous le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir **[zatsit](https://zatsit.fr/)** sur les aspects sociétal et environnemental ; Ce qui m’amène à vous parler du choix sur le matériel qu’a fait la société...  

Reviewer: ```suggestion
On reparle avec tous le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir **[zatsit](https://zatsit.fr/)** sur les aspects sociétal et environnemental.

Ce qui m’amène à vous parler du choix sur le matériel qu’a fait la société.
```

> ## Mon meilleur moment de la journée : la découverte de mon outil de travail...

Reviewer: ```suggestion
## Mon meilleur moment de la journée : la découverte de mon outil de travail
```

> Peu de temps avant mon arrivée, j’ai reçu un message qui me demandait de choisir la configuration de mon futur poste de travail [Framework](https://frame.work/fr/fr)

Reviewer: ```suggestion
Peu de temps avant mon arrivée, j’ai reçu un message qui me demandait de choisir la configuration de mon futur poste de travail [Framework](https://frame.work/fr/fr).
```

J’ai ainsi découvert leur concept : proposer du matériel réparable et évolutif. Ce choix permet à **zatsit** de rester cohérent dans sa politique environnementale et avoir un impact le plus restreint possible sur la planête.

J’ai donc choisi les différents éléments de ma machine : un 13 pouces avec coque en aluminium, le processeur, la ram, le disque, les extensions… Et pour le fun, le montage à faire par mes soins pour prendre en main l'assemblage du hardware.

![Etapes de l'assembalge](framework_assembling.webp)

> Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement et en quelques minutes : on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil qui est fournit, et **zatsit** !

Reviewer: ```suggestion
Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations, et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement et en quelques minutes : on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil qui est fournit, et **zatsit** !
```

> Le petit plus sympa : le contour de l’écran est amovible et il existe plusieurs couleurs, moi j’ai choisi le orange.

Reviewer: ```suggestion
Le petit plus sympa : le contour de l’écran est amovible et il existe plusieurs couleurs. Moi, j’ai choisi le orange.
```

Et tout était fonctionnel au premier démarrage !

J’ai commencé à travailler sur ce poste après avoir configuré un environnent de développement. Je suis enthousiaste à l’idée d'utiliser ma machine lors de mes missions.

Parcourir le site de Framework [Framework](https://frame.work/fr/fr) est intéressant : 
- on y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclé.
- Qu’il est aussi possible de commander tous les composants séparément (cela va jusqu'au kit de visserie) ce qui permet de réparer ou de faire évoluer sa configuration facilement au fil des années (Idée que je trouve intéressante quand on à déjà ouvert un ordinateur portable et de constaté que l’on ne pourra pas faire grand chose pour se dépanner)
- Pour chaque composant proposé sur la marketplace, il existe un tutoriel de remplacement avec la difficulté, les étapes et le temps nécessaire à l'opération. Ce qui permet d'effectuer facilement la maintenance de son poste de travail par ses propres moyens.

![Exemple d'un guide de réparation](reparation.webp)

## En attendant de trouver la mission de mes rêves...

> Avant d'avoir une nouvelle affectation chez un client, durant quelques semaines, j'ai commencé les classroom élaborées par des membres de **zatsit**. Les premiers ateliers m’ont permis de découvrir l’éco-conception, d’étudier et de comprendre les solutions de monitoring pour l’empreinte carbone de projets cloud. La société veut sensibiliser ses collaborateurs aux impacts environnementaux de nos divers métiers. 

Reviewer: ```suggestion
En attendant mon démarrage chez un client, j'en ai profité pour commencer les *Classrooms* élaborées par **zatsit**.

Les premiers ateliers m’ont permis de découvrir les principes de l’éco-conception ou d’étudier et de comprendre les solutions de monitoring pour l’empreinte carbone de projets cloud. 

La société veut sensibiliser ses collaborateurs aux impacts environnementaux de nos métiers. 
```

> Des sujets très intéressants et que j’avais un peu survoler jusqu’à aujourd’hui. Cela va impacter ma façon de concevoir mes futurs projets en tant que développeur. Ces classroom permettent aussi les échanges parmi mes collègues.

Reviewer: ```suggestion
Des sujets très intéressants et que j’avais un peu survolé jusqu’à aujourd’hui. Cela va **impacter** ma façon de concevoir mes futurs projets en tant que technical leader.
```

> ## Mon avenir chez zatsit...

Reviewer: ```suggestion
## Mon avenir chez zatsit ?
```

> Je pense que les mois à venir vont être enrichissants et vont encore me faire grandir.

Reviewer: ```suggestion
Je suis persuadé que les mois à venir vont être enrichissants et vont me faire grandir.
```

> D’une part, l’équipe **zatsit** est à l’écoute de mes souhaits sur le périmètre de mes futures missions et de mes envies de découvrir d’autres écosystèmes chez nos clients, de me donner l’opportunité de faire mes preuves en tant que tech lead.

Reviewer: ```suggestion
D’une part, l’équipe **zatsit** est à l’écoute quant à mes souhaits sur le périmètre de mes futures missions, mes envies de découvrir d’autres écosystèmes chez nos clients, et de me donner l’opportunité de faire mes preuves en tant que tech lead.
```

D’autres part, je vais pouvoir prendre part à des projets en interne toujours en lien avec les convictions de la société qui vont favoriser les échanges au sein d’un collectif de passionnés.

> Pour donner un exemple, on se prépare à participer à l’Hacktoberfest, événement qui encourage les développeurs à contribuer à des projets open source.

Reviewer: ```suggestion

Pour donner un exemple, on se prépare à participer à l’Hacktoberfest, événement qui encourage les développeurs à contribuer sur des projets open source.
```

> Et il se peut que l’on se croise lors d’un BBL ou d’un Meetup sur des sujets passionnants organisé par **zatsit**

Reviewer: ```suggestion
Il se peut d'ailleurs que l’on se croise lors d’un BBL, ou d’un Meetup sur des sujets passionnants organisé par **zatsit**

Alors, prêt à rejoindre l'aventure **zatsit** ?
```

---
slug: onboarding
title: Mon onboarding chez zatsit
authors: [jvasseur]
tags: [general]
---

> Mon arrivée dans une nouvelle aventure !

Reviewer 2: ```suggestion
Mon arrivée dans une nouvelle aventure ! Découvrez les valeurs que j'aime chez **zatsit** et comment se passe l'onboarding.
```

<!-- truncate -->

![Gourde  et sweat](gourde_sweat_banniere.webp)

> Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de mission RH de choc). Un bureau m'attend avec quelques goodies estampillés aux couleurs de l'entreprise : gourde, sweat, sticker et un petit mot d'accueil qui me font très plaisir. 

Reviewer 2: ```suggestion
Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de missions RH de choc). Un bureau m'attend avec quelques goodies estampillés aux couleurs de l'entreprise : gourde en métal recyclé, sweat (dont le fabricant et membre de fairwear.org), sticker et un petit mot d'accueil qui me font très plaisir. 
```

![Etapes de l'assembalge](framework_assembling.webp)

Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations, et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement et en quelques minutes : on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil qui est fournit, et **zatsit** !

Le petit plus sympa : le contour de l’écran est amovible et il existe plusieurs couleurs. Moi, j’ai choisi le orange.

![les extensions et leurs emplacements](extensions.webp)

Pour pouvoir alimenter la machine et la connecter à des périphériques, il y a 4 emplacements qui peuvent recevoir des modules d’extension en tous genres : USB type A ou C, HDMI, Ethernet et autres que l'on peut interchanger, et à chaud.

Et pour certaines extensions, on peut également y ajouter un peu de couleur.

![La machine une fois assemblée](setup.webp)

> Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB que j’avais préparé en avance en suivant les [indications fournies sur le site du fabriquant](https://guides.frame.work/Guide/Ubuntu+22.04+LTS+Installation+on+the+Framework+Laptop+13/109).

Reviewer 2: ```suggestion
Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB 
 en suivant les [indications fournies sur le site du fabriquant](https://guides.frame.work/Guide/Ubuntu+22.04+LTS+Installation+on+the+Framework+Laptop+13/109).
```

> Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB que j’avais préparé en avance en suivant les [indications fournies sur le site du fabriquant](https://guides.frame.work/Guide/Ubuntu+22.04+LTS+Installation+on+the+Framework+Laptop+13/109).

Reviewer 2: Elle s'est perdue dans le déménagement, mais on la fournit normalement, je vais en refaire une.

Author: Elle est bien sur le bureau mais je voulais le faire from scratch

Et tout était fonctionnel au premier démarrage !

J’ai commencé à travailler sur ce poste après avoir configuré un environnent de développement. Je suis enthousiaste à l’idée d'utiliser ma machine lors de mes missions.

Parcourir le site de Framework [Framework](https://frame.work/fr/fr) est intéressant : 

> - on y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclé.

Reviewer 2: ```suggestion
- On y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclés.
```

- Qu’il est aussi possible de commander tous les composants séparément (cela va jusqu'au kit de visserie) ce qui permet de réparer ou de faire évoluer sa configuration facilement au fil des années (Idée que je trouve intéressante quand on à déjà ouvert un ordinateur portable et de constaté que l’on ne pourra pas faire grand chose pour se dépanner)
- Pour chaque composant proposé sur la marketplace, il existe un tutoriel de remplacement avec la difficulté, les étapes et le temps nécessaire à l'opération. Ce qui permet d'effectuer facilement la maintenance de son poste de travail par ses propres moyens.

![Exemple d'un guide de réparation](reparation.webp)

> ## En attendant de trouver la mission de mes rêves...

Reviewer 2: ```suggestion
## En attendant de démarrer mon nouveau projet client...
```

En attendant mon démarrage chez un client, j'en ai profité pour commencer les *Classrooms* élaborées par **zatsit**.

Les premiers ateliers m’ont permis de découvrir les principes de l’éco-conception ou d’étudier et de comprendre les solutions de monitoring pour l’empreinte carbone de projets cloud. 

La société veut sensibiliser ses collaborateurs aux impacts environnementaux de nos métiers. 

Des sujets très intéressants et que j’avais un peu survolé jusqu’à aujourd’hui. Cela va **impacter** ma façon de concevoir mes futurs projets en tant que technical leader.

> 

Reviewer 2: ```suggestion

J'ai pu collaborer à un premier projet, qui m'a fait découvrir certaines solutions de Firebase et Google Cloud Platform, auquel mes collègues pourront contribuer dans les semaines qui arrivent.
```

## Mon avenir chez zatsit ?

Je suis persuadé que les mois à venir vont être enrichissants et vont me faire grandir.

D’une part, l’équipe **zatsit** est à l’écoute quant à mes souhaits sur le périmètre de mes futures missions, mes envies de découvrir d’autres écosystèmes chez nos clients, et de me donner l’opportunité de faire mes preuves en tant que tech lead.

> D’autres part, je vais pouvoir prendre part à des projets en interne toujours en lien avec les convictions de la société qui vont favoriser les échanges au sein d’un collectif de passionnés.

Reviewer 2: ```suggestion
D’autre part, je vais pouvoir participer à des projets internes toujours en lien avec les convictions de la société qui vont favoriser les échanges au sein d’un collectif de passionnés.
```

---

Mon arrivée dans une nouvelle aventure ! Découvrez les valeurs que j'aime chez **zatsit** et comment se passe l'onboarding.

<!-- truncate -->

![Gourde  et sweat](gourde_sweat_banniere.webp)

Lundi ! Arrivée dans les locaux, je suis accueilli par Flavien (un des co-fondateurs) et Océane (notre chargée de missions RH de choc). Un bureau m'attend avec quelques goodies estampillés aux couleurs de l'entreprise : gourde en métal recyclé, sweat (dont le fabricant et membre de [fairwear.org](https://www.fairwear.org)), sticker et un petit mot d'accueil qui me font très plaisir. 

La matinée commence par la partie administrative (ce que tout le monde « adore », n'est ce pas ?)
- les divers documents à fournir et quelques validations d’identité habituelles.
- On active mes accès à [Ekip](https://www.ekip.app/). Je découvre ainsi les tickets restaurants **écoresponsables**.
- Puis avec Océane on passe en revue les accès aux outils qui permettent le suivi pour mes futures missions. D'ailleurs, **zatsit** a mis en place une plateforme dédiée pour l'onboarding des consultants, permettant de mettre en avant les quelques « formations » a dérouler pour être opérationnel. 

> On reparle avec tous le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir **[zatsit](https://zatsit.fr/)** sur les aspects sociétal et environnemental.

Reviewer 3: ```suggestion
On reparle avec tout le monde de ce l’on veut construire à l’avenir et des impacts que veut avoir **[zatsit](https://zatsit.fr/)** sur les aspects sociétal et environnemental.
```

Ce qui m’amène à vous parler du choix sur le matériel qu’a fait la société.

## Mon meilleur moment de la journée : la découverte de mon outil de travail

Peu de temps avant mon arrivée, j’ai reçu un message qui me demandait de choisir la configuration de mon futur poste de travail [Framework](https://frame.work/fr/fr).

J’ai ainsi découvert leur concept : proposer du matériel réparable et évolutif. Ce choix permet à **zatsit** de rester cohérent dans sa politique environnementale et avoir un impact le plus restreint possible sur la planête.

J’ai donc choisi les différents éléments de ma machine : un 13 pouces avec coque en aluminium, le processeur, la ram, le disque, les extensions… Et pour le fun, le montage à faire par mes soins pour prendre en main l'assemblage du hardware.

![Etapes de l'assembalge](framework_assembling.webp)

> Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations, et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement et en quelques minutes : on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil qui est fournit, et **zatsit** !

Reviewer 3: ```suggestion
Donc tout était là, dans un carton contenant les diverses parties de mon ordinateur. Quand on a l’habitude de monter des configurations, et comme tout est suffisamment indiqué sur la carte mère, l'assemblage se fait plutôt facilement et en quelques minutes : on installe le disque SSD et la mémoire, on referme avec le couvercle, qui est le clavier, on serre quelque vis avec l’outil qui est fourni, et **zatsit** !
```

![les extensions et leurs emplacements](extensions.webp)

Pour pouvoir alimenter la machine et la connecter à des périphériques, il y a 4 emplacements qui peuvent recevoir des modules d’extension en tous genres : USB type A ou C, HDMI, Ethernet et autres que l'on peut interchanger, et à chaud.

Et pour certaines extensions, on peut également y ajouter un peu de couleur.

![La machine une fois assemblée](setup.webp)

Une fois tout en place, j’ai installé une distribution Linux (Ubuntu 24.04) à partir d’une clé USB que j’avais préparé en avance en suivant les [indications fournies sur le site du fabriquant](https://guides.frame.work/Guide/Ubuntu+22.04+LTS+Installation+on+the+Framework+Laptop+13/109).

Et tout était fonctionnel au premier démarrage !

J’ai commencé à travailler sur ce poste après avoir configuré un environnent de développement. Je suis enthousiaste à l’idée d'utiliser ma machine lors de mes missions.

Parcourir le site de Framework [Framework](https://frame.work/fr/fr) est intéressant : 
- On y découvre que beaucoup de matériaux constituant les postes de travail sont en bonne partie recyclés.

> - Qu’il est aussi possible de commander tous les composants séparément (cela va jusqu'au kit de visserie) ce qui permet de réparer ou de faire évoluer sa configuration facilement au fil des années (Idée que je trouve intéressante quand on à déjà ouvert un ordinateur portable et de constaté que l’on ne pourra pas faire grand chose pour se dépanner)

Reviewer 3: ```suggestion
- Qu’il est aussi possible de commander tous les composants séparément (cela va jusqu'au kit de visserie) ce qui permet de réparer ou de faire évoluer sa configuration facilement au fil des années (idée que je trouve intéressante quand on à déjà ouvert un ordinateur portable et constaté que l’on ne pourra pas faire grand chose pour la remettre sur pieds)
```

> - Pour chaque composant proposé sur la marketplace, il existe un tutoriel de remplacement avec la difficulté, les étapes et le temps nécessaire à l'opération. Ce qui permet d'effectuer facilement la maintenance de son poste de travail par ses propres moyens.

Reviewer 3: ```suggestion
- Pour chaque composant proposé sur la marketplace, il existe un tutoriel de remplacement avec le niveau de difficulté, les étapes et le temps nécessaire à l'opération. Ce qui permet d'effectuer facilement la maintenance de son poste de travail par ses propres moyens.
```

![Exemple d'un guide de réparation](reparation.webp)

## En attendant de démarrer mon nouveau projet client...

En attendant mon démarrage chez un client, j'en ai profité pour commencer les *Classrooms* élaborées par **zatsit**.

Les premiers ateliers m’ont permis de découvrir les principes de l’éco-conception ou d’étudier et de comprendre les solutions de monitoring pour l’empreinte carbone de projets cloud. 

La société veut sensibiliser ses collaborateurs aux impacts environnementaux de nos métiers. 

Des sujets très intéressants et que j’avais un peu survolé jusqu’à aujourd’hui. Cela va **impacter** ma façon de concevoir mes futurs projets en tant que technical leader.

> J'ai pu collaborer à un premier projet, qui m'a fait découvrir certaines solutions de Firebase et Google Cloud Platform, auquel mes collègues pourront contribuer dans les semaines qui arrivent.

Reviewer 3: ```suggestion
J'ai pu collaborer sur un premier projet, qui m'a fait découvrir certaines solutions de Firebase et Google Cloud Platform, auquel mes collègues pourront contribuer dans les semaines qui arrivent.
```
